src/components/ReloadButton.spec.js

```diff
@@ -0,0 +1,21 @@
+import ReloadButton from './ReloadButton';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+describe('Button', () => {
+  it('renders a button', () => {
+    render(<ReloadButton>Reload</ReloadButton>);
```

Comment:
Your button doesn't accept `children` as props. So this should probably not work.
```suggestion
    render(<ReloadButton />);
```